src/main/webapp/search-forms/editor.tsx

```diff
@@ -0,0 +1,80 @@
+import React, { useState } from 'react'
+import Filter from '../query-filters/filter'
+import { Box, Paper, TextField, Divider } from '@material-ui/core'
+import loadable from 'react-loadable'
+import { memo } from 'react'
+
+let MemoizedVisualizations: any = () => null
+if (typeof window !== 'undefined') {
+  MemoizedVisualizations = loadable({
+    loader: () =>
+      import(//prettier-ignore
+      // @ts-ignore
+      /* webpackChunkName: "visualizations" */ '../workspaces/visualizations').then(
```

Comment:
Not sure if there's a better way. Running formatter was moving my ts-ignore outside the context. We can probably remove these if we extract MemoizedVisualizations to it's own file

Reply:
I think we can refactor this into a common module shared with workspaces.